docs/source/basics.rst

```diff
@@ -140,8 +140,15 @@ the builtins contains a definition like this for ``chr``:
 
     def chr(code: int) -> str: ...
 
-In stubs we don't care about the function bodies, so we use an
-ellipsis instead. That ``...`` is three literal dots!
+The stub files, being for annotation-purposes only, don't include
+redundant information about a function. In the example above, the
+three dot literal (ie. ``...``) is used to make the stub files valid 
+python.  The pass statement can be used as an alternative to ``...``. 
+
+.. _library-stubs-missing:
+
+Missing Stub Files
```

Comment:
It's true that this section is a little longer than others, and it's possible it could be helpful to split it up. This heading doesn't seem right for the whole rest of the section, though -- it fits the next sentence or two but becomes a worse fit lower down.

I spent a couple of minutes just now thinking of alternatives. I could still be convinced by one, but I think there probably isn't a heading that can be just dropped in here and be an improvement without doing more work to reorganize the section.